src/agent/onefuzz/src/triage.rs

```diff
@@ -23,15 +25,25 @@ pub struct TriageCommand {
 }
 impl TriageCommand {
     pub fn new(argv: Vec<String>, env: HashMap<String, String>) -> Result<Self> {
```

Comment:
Since the callers refer to the program as a PathBuf, can this be separated out, rather than converting it to a String in a vec?

Reply:
Yep, we can probably just pass an already-constructed `Command`, and invoke `cmd.envs(env)` in the caller.